component.json

```diff
@@ -13,6 +13,11 @@
         "label": "Secret Access Key",
         "required": true,
         "viewClass": "TextFieldView"
+      },
+      "region": {
```

Comment:
It might make more sense to have this exist as part of the in metadata so that files can be transfered to different buckets within a flow.

Additionally, being able to pre-populate values from a list would be beneficial.

Reply:
Thanks Jacob - this configuration is for region.  Typically a single region is used and we can still use multiple buckets in the flow within a single region.

Reply:
This is similar to the aws CLI.  If you run "aws configure", it prompts for Access ID, Secret, and Region, and saves these in a config file.